### PR TITLE
Feature/uppy own ui

### PR DESCRIPTION
**Pull request by benjaminaudet**

### CODE REVIEW

**Comment by benjaminaudet** [comment]
Pull request is made, but not ready to be merge, still has changes to be done, like i18n etc

File: src/components/project/DMSUploadDocument.vue [hunk 1]
```diff
@@ -46,53 +61,87 @@ export default {
...
   mounted () {
     let baseApiUrl = process.env.BD_API_BASE_URL
     let endpointUpload = baseApiUrl + '/cloud/' + this.cloudId + '/project/' + this.projectId + '/document'
     let token = this.$store.state.oidc.access_token
-    let target = this.target
 
     this.uppy = new Uppy({
-      debug: false,
-      autoProceed: false,
+      debug: true,
```
Comment (Amoki) [inline src/components/project/DMSUploadDocument.vue:104]: debug in production?

File: src/components/project/DMSUploadDocument.vue [hunk 2]
```diff
@@ -46,53 +61,87 @@ export default {
...
+      autoProceed: true,
```
Comment (Amoki) [inline src/components/project/DMSUploadDocument.vue:105]: This change is intentional?

File: src/components/project/UploadFile.vue [hunk 3]
```diff
@@ -87,50 +83,49 @@ export default {
     }
   },
   mounted () {
-    let baseApiUrl = process.env.BD_API_BASE_URL
-    let endpointUpload = baseApiUrl + '/cloud/' + this.cloudId + '/project/' + this.projectId + '/document'
-    let token = this.$store.state.oidc.access_token
-    let target = this.target
-    let options = {
+    const baseApiUrl = process.env.BD_API_BASE_URL
+    const endpointUpload = baseApiUrl + '/cloud/' + this.cloudId + '/project/' + this.projectId + '/document'
```
Comment (Amoki) [inline src/components/project/UploadFile.vue:87]: Template string?

**Review by Amoki — CHANGES_REQUESTED** [review]

**Comment by Amoki** [comment]
https://github.com/bimdata/platform/blob/master/index.html#L10 still import old CSS. https://uppy.io/docs/#With-a-module-bundler may be a solution to load CSS cleanly

**Pull request closed without merging**